sgt-back-end/server.js

```diff
@@ -59,49 +58,21 @@ function evalReqBody(req, res) {
   if (isNaN(score) || Number(score) < 0) {
     res.status(400).json({ error: `${score} is not a valid 'score'. Scores must be positive numbers.` });
     return false;
-  } else {
-    return userInputs;
   }
+  return userInputs;
 }
 
-async function loadGrades(res) {
-  const sql = `
+app.get('/api/grades', async (req, res) => {
+  try {
+    const sql = `
     SELECT *
     FROM "grades";
   `;
-  const result = await db.query(sql);
-  if (result) {
-    return result.rows;
-  } else {
-    return false;
-  }
-}
-
-async function loadGradeId(res, id) {
-  const sql = `
-    SELECT *
-    FROM "grades"
-    WHERE "gradeId" = $1;
-  `;
-  const params = [id];
-  const result = await db.query(sql, params);
-  const grade = result.rows[0];
-  if (grade) {
-    return grade;
-  } else {
-    res.status(404).json({ error: `Cannot find grade with gradeId: ${id}` });
-    return false;
-  }
-}
-
-app.get('/api/grades', async (req, res) => {
-  try {
-    const result = await loadGrades();
-    if (result) {
-      res.json(result);
-    } else {
+    const result = await db.query(sql);
+    if (!result) {
```

Comment:
This actually should never happen. If it can't read the table it will throw. If it can, even if the table is empty, it will return an empty `rows` in the result.

Looks like I missed this in my original review.